source/base/thread_management.cc

```diff
@@ -15,10 +15,9 @@
 
 #include <deal.II/base/thread_management.h>
 
-#include <cerrno>
```

Comment:
Hmm. Lets see where this came from:

b3f43379f95158601afa347430c5ea9f2017f2e7 was the commit that added the `errno.h` header so that we could check a pthread return code:

```cpp
-                                     // start new thread
-    const int error
-      = pthread_create (&tid_list.back(), 0, fun_ptr, fun_data);
+                                     // start new thread. retry until
+                                     // we either succeed or get an
+                                     // error other than EAGAIN
+    int error = 0;
+    do
+      {
+	error = pthread_create (&tid_list.back(), 0, fun_ptr, fun_data);
+      }
+    while ( ! ((error == 0) || (error != EAGAIN)));
```


We removed this in 73f135d76d9b79f681e42cda4e5a0c1f9260d5b1 in favor of calling boost functions for handling threads. I believe that we should have removed the header there. I don't see any other error codes in this file and things compile without it.